assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -8,6 +20,64 @@ import './style.scss';
  * Component that renders the Cart block when user has something in cart aka "full".
  */
 const Cart = () => {
+	// @todo this are placeholders
+	const onActivateCoupon = ( couponCode ) => {
+		// eslint-disable-next-line no-console
+		console.log( 'coupon activated: ' + couponCode );
+	};
+	const totalItems = [
+		{
+			label: __( 'List items:', 'woo-gutenberg-products-block' ),
+			value: '€300.00',
+		},
+		{
+			label: __( 'Subtotals', 'woo-gutenberg-products-block' ),
+			value: '€270.00',
+		},
+		{
+			label: __(
+				'Estimated shipping cost:',
+				'woo-gutenberg-products-block'
+			),
+			value: '€6.00',
+			description: __(
+				'Shipping to location (change address)',
+				'woo-gutenberg-products-block'
+			),
+		},
+	];
+	const shippingOptions = [
```

Comment:
Don’t we want to mimic the same behaviour we have at the checkout section?

Reply:
Absolutely! I think when I started working on this branch checkout was not yet merged, that's why I missed it.

I created a placeholder file in 8ccf09b3625ca859e982a5207922d2fb49ae27f6. We are displaying the options a little bit different depending on the screen, so I renamed the properties to make them easier to reuse in different situations.